lib/vtls/schannel.c

```diff
@@ -669,14 +673,20 @@ schannel_connect_step1(struct connectdata *conn, int sockindex)
       CertFreeCertificateContext(client_certs[0]);
 
     if(sspi_status != SEC_E_OK) {
-      if(sspi_status == SEC_E_WRONG_PRINCIPAL)
-        failf(data, "schannel: SNI or certificate check failed: %s",
```

Comment:
Why was this conditional error message removed here?

Reply:
See also #2950

Reply:
Are you saying that failf() was useful there and we should reinstate it?

Reply:
As mentioned in the commit message, `AcquireCredentialsHandle` doesn't return SEC_E_WRONG_PRINCIPAL https://msdn.microsoft.com/en-us/library/windows/desktop/aa374716(v=vs.85).aspx. So I think the previous error message was dead code, unless there is a legacy use case that I overlooked...